src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import style from './style.scss';
+
+const cx = classNames.bind(style);
+
+class Progressbar extends React.Component {
+  constructor (props) {
+    super(props);
+    const { percentage, showPercentageNum, className } = props;
+
+    this.percentage = percentage || 0;
+    this.showPercentageNum = showPercentageNum;
+    this.className = className;
```

Comment:
this is not react style code. To use the `percentage`, `showPercentageNum`, and `className`, you can just use from props. You don't need to assign them to another instances.

Reply:
I am trying to shorten the variable name. I don't want to use the long name like `this.props. percentage`. I failed because I used ES6 class. I will change `class` to `function`.